deps/snappy/freebsd/snappy-stubs-public.h

```diff
@@ -76,10 +80,21 @@ typedef unsigned long long uint64;
 
 typedef std::string string;
 
+#ifndef DISALLOW_COPY_AND_ASSIGN
 #define DISALLOW_COPY_AND_ASSIGN(TypeName) \
   TypeName(const TypeName&);               \
   void operator=(const TypeName&)
+#endif
+
+#if !0
+// Windows does not have an iovec type, yet the concept is universally useful.
+// It is simple to define it ourselves, so we put it inside our own namespace.
+struct iovec {
+	void* iov_base;
+	size_t iov_len;
+};
+#endif
```

Comment:
This struct seems to be in headers for multiple os'es (see linux, mac and win32 headers below) and also mentioning windows. I don't want to be too nit picky, but since it seems to be identical, shouldn't it be refactored out?